entries/entry01.md

```diff
@@ -1,7 +1,32 @@
 # Entry 1
```

Comment:
Looking forward to seeing what you and Osvaldo create! Missing skills and sources.